app/models/artist.rb

```diff
@@ -4,7 +4,5 @@ class Artist < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :works
-  has_many :requests
-  has_many :buyers, through: :requests
```

Comment:
This is a really clever solution! One part that I'm not entirely sure about is whether there is supposed to be another relation in the Artist table. None of the standard relations really make sense, so leaving it like this makes sense to me too. What do you think?

Reply:
hm i see what u mean - i'll think abt it some more